dod/src/main/java/com/revolut/rxdata/dod/DataObservableDelegate.kt

```diff
@@ -71,22 +73,34 @@ class DataObservableDelegate<Params : Any, Domain : Any> constructor(
                 }
                 .toObservable()
                 .subscribeOn(Schedulers.io())
-                .takeIf { memoryIsEmpty } ?: Observable.just(Data(content = memCache)))
+                .takeIf { memoryIsEmpty } ?: just(Data(content = memCache)))
                 .flatMap { cachedValue ->
                     if (forceReload || memoryIsEmpty) {
                         val data = cachedValue.copy(loading = true)
                         subject.onNext(data)
```

Comment:
Should this be removed? Seems it will be emitted on concat?

Reply:
No, this subject might have other subscribers, concat will emit to the current one